beacon-chain/powchain/service.go

```diff
@@ -172,17 +174,17 @@ func NewService(ctx context.Context, config *Web3ServiceConfig) (*Service, error
 		config.Eth1HeaderReqLimit = defaultEth1HeaderReqLimit
 	}
 
-	config.HTTPEndpoints = dedupEndpoints(config.HTTPEndpoints)
+	config.HttpEndpoints = dedupEndpoints(config.HttpEndpoints)
```

Comment:
It would be better to dedup the string values first, otherwise you could inadvertently end up regarding 2 endpoints with different auth tokens as the same.

Reply:
`dedupEndpoints` looks at the URL to determine equality. But if I move parsing into `powchain`, then I can definitely dedup strings. It will be easier that way.